UnityAds/src/main/java/com/mopub/mobileads/UnityRouter.java

```diff
@@ -67,6 +74,11 @@ static void initMediationMetadata(Context context) {
         mediationMetaData.setVersion(MoPub.SDK_VERSION);
         mediationMetaData.set("adapter_version", UnityAdsAdapterConfiguration.ADAPTER_VERSION);
         mediationMetaData.commit();
+
+        MetaData headerBiddingMeta = new MetaData(context);
+        headerBiddingMeta.setCategory("headerbidding");
+        headerBiddingMeta.set("mode", "enabled");
+        headerBiddingMeta.commit();
```

Comment:
Are we expecting to have HB enabled by default? Might want to remove this if it was just for testing purposes